cli/cli.go

```diff
@@ -94,6 +94,70 @@ func main() {
 				},
 			},
 		},
+		{
+			Name: "delete",
+			Subcommands: []cli.Command{
+				{
+					Name:      "pipeline",
+					Usage:     "delete pipeline",
+					ArgsUsage: "<pipeline-name>",
+					Before:    requireNameArg,
+					Action:    deletePipeline,
+				},
+				{
+					Name:      "build",
+					Usage:     "delete build",
+					ArgsUsage: "<pipeline-name>",
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:  "build, b",
+							Usage: "build number, if not provided will not proceed deletion",
+						},
+					},
+					Before: requireNameArg,
+					Action: deleteBuild,
+				},
+			},
+		},
+		{
+			Name:  "deploy",
+			Usage: "deploy kontinuous app in the cluster",
+			Subcommands: []cli.Command{
+				{
+					Name:   "remove",
+					Usage:  "remove kontinuous app in the cluster",
+					Action: removeDeployedApp,
+				},
+			},
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "namespace",
+					Usage: "Required, kontinuous namespace",
```

Comment:
`default` is usually polluted with other stuff. We'd need to create a `kontinuous` namespace but would mean its easier to manage (and the namespace can just be deleted to clean up)